### PR TITLE
Show triggers in dalstatement.g4

### LINKED ISSUE #13599: Show triggers is misspelled in dalstatement.g4
## Bug Report

I found **show triggers** is misspelled in dalstatement.g4

### PR DESCRIPTION
**Pull request by Swastyy**
Fixes #13599 

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4 [hunk 1]
```diff
@@ -234,8 +234,8 @@ showStatus
     : SHOW (GLOBAL | SESSION)? STATUS showFilter?
     ;
 
-showTrriggers
-    : SHOW TRIGGER fromSchema? showFilter?
+showTriggers
+    : SHOW TRIGGERS fromSchema? showFilter?
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4:238]: @Swastyy Can you add some parse test for this change?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4 (hunk 1 above)
Comment (Swastyy) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4:238]: Is there any example PR which can be a reference for it?

Comment (strongduanmu) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4:238]: > Is there any example PR which can be a reference for it?

@Swastyy You can refer pr #13446.

**Comment by strongduanmu** [comment]
Hi @Swastyy, any progress?

**Comment by Swastyy** [comment]
> Hi @Swastyy, any progress?

Hi @strongduanmu I have an exam so I was busy, I will try completing it in 2 to 3 days.

**Comment by strongduanmu** [comment]
> as busy, I will try completing it

@Swastyy Okay, waiting for your good news.

**Comment by strongduanmu** [comment]
@Swastyy This issue has been fixed by #13629, so I will close this PR. You are very welcome to continue participating in other tasks in the community.

**Pull request closed without merging**